tests/plugins/colorbutton/manual/colorhistory/4-colororder.md

```diff
@@ -0,0 +1,58 @@
+@bender-tags: 4.15.0, feature, 1795
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, colorbutton, colordialog, sourcearea, removeformat, undo
+
+**Note:** At the beginning open console and watch for errors (especially when picking colors).
+
+1. Click on color button.
+
+  **Expected:**
+
+  1. Below default palette there is horizontal rule and color history row.
+  1. There are 4 color boxes in color history row.
+  1. Color boxes are ordered by number of occurences and further by order in document:
+      * Strong Blue (occures 2 times in document)
+	  * Red (occures 1 time and is 1st color in doc)
+	  * Grey (occures 1 time and is 2nd color in doc)
+	  * Yellow (occures 1 time and is 3rd color in doc)
```

Comment:
Maybe we could be less specific about the color position and its occurrences, so basically removing everything inside brackets as it's not really needed to finish the test?